tests/unit/db/result_test.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# Copyright (c) 2015 IBM. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+result module unit tests
+"""
+import unittest
+
+from cloudant.design_document import DesignDocument
+from cloudant.errors import CloudantArgumentError
+from cloudant.result import python_to_couch, Result
+from cloudant.views import View
+from tests.unit.db.unit_t_db_base import UnitTestDbBase
+
+
+class PythonToCouchTests(unittest.TestCase):
+    """
+    test cases for validating python_to_couch's options
+    """
+
+    def test_valid_option_group_level(self):
+        """
+        Test group_level option is valid
+        """
+        opts = {
+            "group_level": 100
+        }
```

Comment:
Another minor nit here.  This is a pretty simple dictionary, I think it can be all on one line `opts= {'group_level': 100}`.  Also, while in Python it technically does not matter if you use a single quote or a double quote for strings, I have tried to use singles throughout, I know that the tests in the mocked directory sort of deviate from this but they need to be cleaned up at some point to match.